[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\65605\\AppData\\Local\\Temp\\quarto-sessioncc965f4fdd13a1ee'\n"]
